### PR TITLE
Release 5 2023_10_11

### PR DESCRIPTION
**Pull request by FooRider**

### CODE REVIEW

File: sf-docker/all-in-one/docker-compose.yml [hunk 1]
```diff
@@ -542,11 +594,39 @@ services:
...
       - AppSettings__Log_RabbitMq_Enabled
       - NoSqlDataStorageDisabled
+      - OperationMode__MatchingDataStrategy
+      - OperationMode__LivenessDataStrategy
+    volumes:
+      - "./iengine.lic:/etc/innovatrics/iengine.lic"
+
+  edge-streams-state-synchronizer:
+    image: ${REGISTRY}sf-edge-streams-state-synchronizer:${SF_VERSION}
+    container_name: SFEdgeStreamsStateSynchronizer
```
Comment (blaskoa) [inline sf-docker/all-in-one/docker-compose.yml:611]: missing `restart` policy

File: sf-docker/all-in-one/create-wl-stream-generation.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+source .env
```
Comment (blaskoa) [inline sf-docker/all-in-one/create-wl-stream-generation.sh:3]: this did not work on test server

Comment (FooRider) [reply sf-docker/all-in-one/create-wl-stream-generation.sh]: Confirmed, fixed now with a bash function for getting the values from .env file.

File: sf-docker/all-in-one/docker-compose.yml (hunk 1 above)
Comment (FooRider) [reply sf-docker/all-in-one/docker-compose.yml:611]: added

File: sf-docker/all-in-one/create-wl-stream-generation.sh [hunk 3]
```diff
@@ -1,10 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
-source .env
+getvalue() {
+    local key="$1"
+    local value=$(grep -E ^${key}= .env | cut -d '=' -f2- | cut -d$'\r' -f1)
```
Comment (blaskoa) [inline sf-docker/all-in-one/create-wl-stream-generation.sh:5]: there are some very minor differences between this function and the grep from .env in `run.sh` script, why is that?

also maybe we can reuse the function in both scripts

**Review by blaskoa — APPROVED** [review]

File: sf-docker/all-in-one/create-wl-stream-generation.sh (hunk 3 above)
Comment (FooRider) [reply sf-docker/all-in-one/create-wl-stream-generation.sh:5]: This version can handle values containing char '=', the run.sh one can't handle them (thus problems with connection string).
I'll fix the run.sh one.

**Review by Temppus — APPROVED** [review]

**Pull request merged**